e2e/next/src/next.test.ts

```diff
@@ -318,7 +319,7 @@ describe('Next.js Applications', () => {
     });
   }, 120000);
 
-  it('should build with public folder', async () => {
+  it.only('should build with public folder', async () => {
```

Comment:
this is ok?

Reply:
still testing some things out, still in draft 😀